docs/intro/install.md

```diff
@@ -9,7 +9,7 @@ compiled releases can be manually downloaded and installed.
 
 1. Download your desired version from [the releases
    page](https://github.com/deislabs/krustlet/releases)
-1. Unpack it (`tar -xzf krustlet-v0.1.0-Linux-amd64.tar.gz`)
+1. Unpack it (`tar -xzf krustlet-v0.1.0-linux-amd64.tar.gz`)
```

Comment:
This shouldn't change until we release 0.2 as the old one still has the uppercase

Reply:
fair enough!